packages/frontend-app/src/app/components/Card/Card.tsx

```diff
@@ -23,7 +28,16 @@ export const Card: React.FC<IProps> = ({ src, title, artist, link }) => {
             <IconButton>
               <PlayIcon size={13} className="ml-1" />
             </IconButton>
-            <IconButton>
+            <IconButton
+              onClick={() => {
+                copy(link as string);
```

Comment:
```suggestion
                copy(link);
```
You don't need to force the type to `string` because link is already a string